components/form/overview.md

```diff
@@ -162,26 +152,40 @@ The component reference provides you with access to the `EditContext` object tha
 }
 ````
 
-## Automatic Generation of fields
+## Building Blocks
```

Comment:
This section is large and has 3 focal points, instead of one (e.g. parts of the form, data annotation attributes, automatic generation + customizations). Consider splitting into several sections that are simpler and more focused.